### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by Andrecast**
Please, check my code

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,17 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Returns the object based on the class and its ID"""
+        objects = self.all(cls).values()
+        object_id = [item for item in objects if item.id == id][0]
```
Comment (vicuartas230) [inline models/engine/db_storage.py]: I am watching you add the index of the list comprehension before the statement and that is going to launch an error. Please check when the Id doesn't exist.

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -74,3 +74,17 @@ def reload(self):
...
+        if object_id:
+            return object_id
+        else:
```
Comment (vicuartas230) [inline models/engine/db_storage.py]: You can remove this else in order to the code be shorter

**Review by vicuartas230 — CHANGES_REQUESTED** [review]

File: models/engine/db_storage.py (hunk 1 above)
Comment (Andrecast) [reply models/engine/db_storage.py]: I already changed the index before the statement

File: models/engine/db_storage.py (hunk 2 above)
Comment (Andrecast) [reply models/engine/db_storage.py]: I removed the else condition and put the return instead

**Review by vicuartas230 — APPROVED** [review]
You did it. Well done!

**Pull request merged**